.github/workflows/gradle-plugin.yml

```diff
@@ -17,8 +17,8 @@ jobs:
       fail-fast: false
       matrix:
         os: ['ubuntu-24.04', 'macos-14', 'windows-2022']
-        gradle: ['8.7', '8.13']
-        agp: ['8.6.0', '8.9.0']
+        gradle: ['8.7', '9.0.0']
```

Comment:
Looks like it fails on 9.0.0. If it is unrelated, we have to remove it from this PR, fix the 9.0.0 compatibility, and then return